classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :student
```

Comment:
This instance attribute seems not to be initialized or used. I think you meant `@students` and not `@student`. Please, consider correcting it.